### PR TITLE
Corrected erroneous 'Buy Reward' sound - fixes #6668

### LINKED ISSUE #6668: Plays wrong sound when cannot buy item due to insufficient gold
My User ID: 6d9d36a8-d91f-4737-8e8c-4fe9a4d632fe
I'm using Windows 7, Chrome 48.0.2564.103 m.

When you try to buy an item when you don't have enough gold, it still plays the "success" sound as if you bought it, even though the "Not Enough Gold" message appears in the upper right corner of the browser window.

The expected behavior for this is to play a different sound, just like the sounds for "+" and "-" on a habit are different.  The sound should indicate that the transaction was not completed (something similar to the "lost health" sound.

### PR DESCRIPTION
**Pull request by kdcarlsen**
Simply added if statements to prevent calls to function that plays reward sound if the user does not have a gp balance of at least the cost of the equipment (or reward/task).

User ID:
ac0b4f0d-4b10-4073-a1fe-308fd20900ee

### CODE REVIEW

File: website/public/js/controllers/tasksCtrl.js [hunk 1]
```diff
@@ -221,7 +224,11 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
 
     $scope.buy = function(item) {
       User.user.ops.buy({params:{key:item.key}});
-      $rootScope.playSound('Reward');
+      //Plays sound for built in rewards (armor, potions etc)
+      if(item.value <= User.user.stats.gp){
+        $rootScope.playSound('Reward');
+    }
```
Comment (crookedneighbor) [inline website/public/js/controllers/tasksCtrl.js]: The indentation on this brace is incorrect.

Comment (kdcarlsen) [inline website/public/js/controllers/tasksCtrl.js]: Oops, you're right!  Clearly I was coding too long yesterday - I'll fix now and recommit/push.

**Comment by kdcarlsen** [comment]
Fixed brace error

**Comment by kdcarlsen** [comment]
Any updates on this?  I fixed the bracing error - are there any additional edits I need to make?

**Comment by crookedneighbor** [comment]
Sorry about the delay. We've all been busy with other portions of the site. Looking at this now.

**Comment by crookedneighbor** [comment]
Merged in with https://github.com/HabitRPG/habitrpg/commit/7b708f51ba240d20cd0e3df44fe19c90172d4806

Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Pull request closed without merging**